packages/notification-center/src/components/notification-center/components/AppContent.tsx

```diff
@@ -19,47 +21,42 @@ export function AppContent() {
     }
   );
 
+  const primaryColor = organization?.branding?.color ?? theme.loaderColor;
   const themeConfig = {
-    colors: {
-      main: theme.loaderColor || organization?.branding?.color,
-      secondaryFontColor: theme.layout?.wrapper.secondaryFontColor,
-    },
+    primaryColor: 'brand',
```

Comment:
the Mantine `primaryColor` is a key from the `colors` array, it is then used on the UI components. But I didn't notice any effect of this because we do override colors, backgrounds, etc.
Now, I've removed the `styled-components` lib, to which we were passing the theme config.
Mantine + Emotion work well together, but we just needed to wrap everything by `MantineProvider` and pass the new theme config.
Finally because before the primary color was referring to our brand's primary color, I've made corresponding changes in the Mantine config.